tests/data/config/severalrules.yaml

```diff
@@ -0,0 +1,13 @@
+disabled_sal_components:
+- ATCamera
+rules:
+- class: Heartbeat
+  configs:
+  - name: ATPtg
```

Comment:
ATPtg is not outputting heartbeats yet. Probably a good idea to remove it or now.

Reply:
This is for unit tests and will not rely on the real ATPtg running, so it won't harm anything. But there is no point setting a bad example so I will change it.